src/03-filter.js

```diff
@@ -13,21 +13,39 @@ const exampleSongData = require("../data/songs");
  * @param {Object[]} songs - An array of songs. See the song data for more.
  * @returns {Object[]} An array of objects.
  */
-function getSongsBySaib(songs) {}
+function getSongsBySaib(songs) {
+  return songs.filter((song) => {
+    if (song.artist === "Saib") {
+      return song;
+    }
+  });
+}
```

Comment:
`.filter` looks for its callback to return `true` or `false`. `return song` works because each `song` is truthy, but your code will follow the standards a lot better if you return `true` instead.

As a very succinct version of this, you could also do `return song.artist === 'Saib'` instead of the `if`!